tensorboard/webapp/testing/utils.ts

```diff
@@ -57,18 +57,41 @@ import {
   createSettingsState,
 } from '../settings/testing';
 import {HPARAMS_FEATURE_KEY} from '../hparams/_redux/types';
+import {ALERT_FEATURE_KEY, AlertState} from '../alert/store/alert_types';
+import {
+  PERSISTENT_SETTINGS_FEATURE_KEY,
+  PersistentSettingsState,
+} from '../persistent_settings/_redux/persistent_settings_types';
+import {SETTINGS_FEATURE_KEY} from '../settings/_redux/settings_types';
+import {CORE_FEATURE_KEY} from '../core/store/core_types';
+import {
+  DEBUGGER_FEATURE_KEY,
+  DebuggerState,
+} from '../../plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_types';
+
+interface FullState extends State {
+  [DEBUGGER_FEATURE_KEY]: DebuggerState;
+  [ALERT_FEATURE_KEY]: AlertState;
+  [PERSISTENT_SETTINGS_FEATURE_KEY]: PersistentSettingsState;
+}
 
-export function buildMockState(overrides: Partial<State> = {}): State {
+export function buildMockState(overrides: Partial<FullState> = {}): State {
```

Comment:
Previously this did not support overriding all the different kinds of state because the overrides were not being used. They still aren't all being used but making this fully capable seemed like a good thing to do.